cmake/app/boilerplate.cmake

```diff
@@ -17,7 +17,7 @@
 # It exists to reduce boilerplate code that Zephyr expects to be in
 # application CMakeLists.txt code.
 
-# CMake version 3.13.1 is the real minimum supported version.
+# CMake version 3.20 is the real minimum supported version.
```

Comment:
The CMake version required here is the real minimum supported version.

Then we don't need to keep bumping this.

cmake_minimum_required(VERSION 3.20.0)
is self documenting.

Reply:
yes, but this sentence should be read together with the text below:

https://github.com/zephyrproject-rtos/zephyr/blob/8482710598345e0ea36b5eaf72c4b453ef2d8951/cmake/app/boilerplate.cmake#L20-L25

In which case I would say it makes sense to mention 3.20.
Removing the sentence with 3.20 and keeping the rest makes the whole comment look strange.

A proposal with a complete re-write of the whole comment section would be accepted, but for a start I just propose bumping the version instead of a re-write.